readme.md

```diff
@@ -50,11 +59,13 @@ What's wrong with this mongoose code and how might we fix it?
 ```js
 var results = AuthorModel.find({name: "Bob"});
 console.log(results);
+
 ```
 
 Your Answer:
 ```text
 
+First, we are using .find versus .findOne method in mongoose. The .find method will return an array of documents, where as .findOne will return a single document. Additionally, we might want to pass an additional callback function in order to grab the results.
```

Comment:
That's correct. The main thing we were looking for is that the results are only valid in a callback function.